Cargo.toml

```diff
@@ -34,6 +34,7 @@ slog = "^2.7.0"
 tempfile = "3.5.0"
 tokio = "^1.36.0"
 url = "^2.4.1"
+ic-sns-cli = { git = "https://github.com/dfinity/ic", rev = "5fe907da2193c6051634b29133280f53f2490d52" }
```

Comment:
> What is the plan to keep this up to date? 

There is no plan :D however it is not a regression since before this MR we were in the same situation (having to manually remember to update a git commit hash in this repository, which we basically never did).

Reply:
dfx uses a particular version of the replica, which we typically update before making a dfx release, sometimes more often.
It's here: https://github.com/dfinity/sdk/blob/master/nix/sources.json#L158
By default, that rev is a commit that has been proposed and executed.  We get the rev from
```
curl "https://ic-api.internetcomputer.org/api/v3/subnet-replica-versions?limit=50&offset=0" | jq '.data[0].replica_version_id'
```

Reply:
For completeness, we update this rev by running [this workflow](https://github.com/dfinity/sdk/blob/master/.github/workflows/update-replica-version.yml), which creates a PR like [sdk #3675](https://github.com/dfinity/sdk/pull/3675)

Reply:
> Something closer to my suggestion, but more readily implementable: rev = "master". I guess what they can do whenever there is a breakage (should be rare) is temporarily "freeze" this. That is, they would temporarily go back to rev = "specific-git-hash" until master has a fix that allows them to go back to rev = "master".

This will get frozen to a particular commit in `Cargo.lock`. It could be upgraded with `cargo upgrade` but that would still have to be done manually.

@ericswanson-dfinity When you update the rev for dfx, will you also update it in dfx-extensions?

Reply:
No, the SDK team does not maintain the sns or nns extensions